numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1333,6 +1333,138 @@ def test_zipf(self):
         assert_array_equal(actual, desired)
 
 
+class TestMultivariateHypergeometric(object):
+
+    def setup(self):
+        self.seed = 123456789
+
+    def set_seed(self):
+        random.bit_generator.seed(self.seed)
```

Comment:
This method no longer exists. Use a fresh `BitGenerator` with the given seed each time instead of a single global that gets reset.

Reply:
Fixed in the latest update.